AK/JsonParser.cpp

```diff
@@ -231,6 +232,30 @@ ErrorOr<JsonValue> JsonParser::parse_number()
         break;
     }
 
+#ifndef KERNEL
```

Comment:
Do we want to support this parsing in for all JSON?

Reply:
Since this is the general JsonParser, exponents are now supported everywhere in SerenityOS. 
So the question is really is it fine to support exponents everywhere in Serenity?

Reply:
yes, as per the JSON spec :)